devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java

```diff
@@ -345,10 +347,7 @@ public void startDev() {
             pb.redirectErrorStream(true);
             pb.redirectInput(ProcessBuilder.Redirect.INHERIT);
             pb.directory(getWorkingDir());
-            System.out.println("Starting process: ");
-            pb.command().forEach(System.out::println);
-            System.out.println("Args: ");
-            args.forEach(System.out::println);
+            getLogger().lifecycle("Launching JVM with command line: " + pb.command().stream().collect(joining(" ")));
```

Comment:
Is it normal that we don't have the args anymore?

Reply:
Yes, it shows the same values as `pb.command()`. See https://github.com/quarkusio/quarkus/pull/6057/files#diff-72fac779acece3ea68a6666ee0060ad5L344